subprojects/platform-native/src/testFixtures/groovy/org/gradle/nativeplatform/fixtures/AvailableToolChains.java

```diff
@@ -109,6 +114,9 @@ public static InstalledToolChain getToolChain(ToolChainRequirement requirement)
      * @return A list of all known tool chains for this platform. Includes those tool chains that are not available on the current machine.
      */
     public static List<ToolChainCandidate> getToolChains() {
+        if (MAC_OS_X_M1.isFulfilled()) {
```

Comment:
Why should we return no toolchains on mac M1?

Reply:
There was something wrong unresolved when searching for available toolchains on M1: 

```
java.lang.RuntimeException: Could not execute /usr/bin/xcodebuild. Error: xcrun: error: unable to load libxcrun (dlopen(/opt/xcode/Xcode_10.1.app/Contents/Developer/usr/lib/libxcrun.dylib, 0x0005): could not use '/opt/xcode/Xcode_10.1.app/Contents/Developer/usr/lib/libxcrun.dylib' because it is not a compatible arch).
, Output: 
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:72)
	at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:59)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
	at org.gradle.test.fixtures.file.TestFileHelper.executeSuccess(TestFileHelper.groovy:209)
	at org.gradle.test.fixtures.file.TestFile.execute(TestFile.java:868)
	at org.gradle.nativeplatform.fixtures.AvailableToolChains.lambda$findXcodes$7(AvailableToolChains.java:741)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
```